front/src/pages/Tags.tsx

```diff
@@ -97,30 +107,219 @@ class Tags extends Component<Props, State> {
+                      onClick={() => {
+                        this.setState({
+                          parent_id: tag.id
+                        });
+                        this.addTag();
+                      }}
+                    >
+                      <Icon type='plus' />
+                    </a>
+                  )}
+                  &nbsp;&nbsp;
+                  {tag.is_active && (
+                    <a
+                      className='disable_tag'
+                      onClick={() => {
+                        this.disableTag(tag.id);
+                      }}
+                    >
+                      <Icon type='stop' />
+                    </a>
+                  )}
+                  {!tag.is_active && (
+                    <a
+                      className='enable_tag'
+                      onClick={() => {
+                        tag.is_active = true;
+                        this.enableTag(tag);
+                      }}
+                    >
+                      <Icon type='check' />
+                    </a>
+                  )}
+                </div>
+              }
+            >
               {loop(tag.children)}
             </TreeNode>
           );
         }
-        return <TreeNode key={String(tag.id)} title={tag.name} />;
+        return (
+          <TreeNode
+            className='tag_wrapper'
+            key={String(tag.id)}
+            title={
+              <div>
+                <span
+                  className={tag.is_active ? 'tag_title' : 'tag_title_disabled'}
+                >
+                  {tag.name}&nbsp;&nbsp;
+                </span>
+                {tag.is_active && (
+                  <a
+                    className='add_tag'
+                    onClick={() => {
+                      this.setState({
+                        parent_id: tag.id
+                      });
+                      this.addTag();
+                    }}
+                  >
+                    <Icon type='plus' />
+                  </a>
+                )}
+                &nbsp;&nbsp;
+                {tag.is_active && (
+                  <a
+                    className='disable_tag'
+                    onClick={() => {
+                      this.disableTag(tag.id);
+                    }}
+                  >
+                    <Icon type='stop' />
+                  </a>
+                )}
+                {!tag.is_active && (
+                  <a
+                    className='enable_tag'
+                    onClick={() => {
+                      tag.is_active = true;
+                      this.enableTag(tag);
+                    }}
+                  >
+                    <Icon type='check' />
+                  </a>
+                )}
+              </div>
+            }
+          />
+        );
       });
 
     return [
       <Tree
         key={1}
         className='draggable-tree'
-        draggable={false}
+        draggable={true}
```

Comment:
Nope, doit être désactivé, c'est pas géré jusqu'au bout :)
```suggestion
        draggable={false}
```